openjdk-11/Dockerfile

```diff
@@ -3,5 +3,12 @@ FROM openjdk:11.0.2-jre-slim-stretch
 LABEL maintainer="gs-w_eto_eb_federal_employees@usgs.gov"
 
 ENV USER=java
+ENV HOME=/home/$USER
 
 RUN adduser --disabled-password --gecos "" -u 1000 $USER
+
+WORKDIR $HOME
+
+COPY --chown=1000:1000 docker-entrypoint.sh docker-entrypoint.sh
+
+RUN chmod +x docker-entrypoint.sh
```

Comment:
What are your thoughts about including `ENTRYPOINT ["./docker-entrypoint.sh"]` in the base image. That way, the environment variable can always happen without having to specified in an application's Dockerfile. 

I'm still weighing the pros/cons. Thoughts?

Reply:
Including the ENTRYPOINT allows the removal of the launch-app.sh script. Let's get some experience and see how the ENTRYPOINT here works for us.